.github/workflows/flutter.yml

```
@@ -0,0 +1,208 @@
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  setup:
+    name: Setup Environment
+    runs-on: ubuntu-latest
+    outputs:
+      flutter_version: ${{ steps.flutter-version.outputs.FLUTTER_VERSION }}
+      coverage_excludes_regex_flags: ${{ steps.extract-config.outputs.COVERAGE_EXCLUDES_REGEX_FLAGS }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)